types/video.js/index.d.ts

```diff
@@ -453,6 +453,19 @@ declare namespace videojs {
      */
     function use(type: string, middleware: (player: Player) => Middleware): void;
 
+    /**
+     * Used to subclass an existing class by emulating ES subclassing using the extends keyword.
+     * @param superClass super component to extend
+     * @param subClassMethods methods sub class will add to super
+     */
+    function extend<
+        TSuper extends new (...args: any[]) => any,
+        TSubClassMethods extends Record<string | symbol, (this: InstanceType<TSuper>, ...args: any[]) => any>,
+    >(
+        superClass: TSuper,
+        subClassMethods: TSubClassMethods,
```

Comment:
```suggestion
        subClassMethods?: TSubClassMethods,
```
https://docs.videojs.com/extend.js.html#line29